### ISSUE TITLE
failed to alloc memory error on Windows 10

### ISSUE THREAD

**Issue by andreymoskvenkov** [funakoshi2718/funakoshi-miner#41]
Mining rig: 8x 1080 ti (7x Asus Strix 1080 ti, 1x Asus Strix 1080 ti OC edition).
Both are overclocked, using MSI Afterburner.
Nvidia drivers: latest.
OS: Windows 10 x64 Pro.
BWF's miner works without any issues in such configuration.

Miner version: 4.8

Command:  funakoshiMiner.exe -cd 0 1 2 3 4 5 6 7 --algo 144_5 --pers BgoldPoW -l btg.2miners.com:4040 -u GXd3AQrThEmB5oUDqwcVw798PZ5u2Xrfdc.rig1 --pass x

[2018-Sep-19 11:29:20] [MINER] Using pool: btg.2miners.com
[2018-Sep-19 11:29:20] [MINER] Starting miner
[2018-Sep-19 11:29:20] [GPU-0] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 0)
[2018-Sep-19 11:29:20] [GPU-1] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 1)
[2018-Sep-19 11:29:20] [GPU-2] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 2)
[2018-Sep-19 11:29:20] [GPU-3] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 3)
[2018-Sep-19 11:29:20] [GPU-4] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 4)
[2018-Sep-19 11:29:20] [GPU-5] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 5)
[2018-Sep-19 11:29:20] [GPU-6] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 6)
[2018-Sep-19 11:29:20] [MINER] Connecting to stratum server btg.2miners.com:4040
[2018-Sep-19 11:29:20] [GPU-7] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 7)
[2018-Sep-19 11:29:20] [MINER] Connected!
[2018-Sep-19 11:29:20] [MINER] Subscribed to stratum server
[2018-Sep-19 11:29:20] [MINER] Authorized worker GXd3AQrThEmB5oUDqwcVw798PZ5u2Xrfdc.rig1
[2018-Sep-19 11:29:20] [MINER] Target set to 000f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0d
[2018-Sep-19 11:29:20] [MINER] Received new job #870787144854837
[2018-Sep-19 11:29:21] [GPU-4] CUDA: failed to alloc memory

**Comment by funakoshi2718** [comment]
Hi,
Can you please try to reduce OC and see what happen.
Each miner software requires different OC  setup, because they access memory differently.
Maybe the OC is too much.

**Comment by andreymoskvenkov** [comment]
No, I won't reduce overclocking settings. EWFB's miner works fine with any OC profiles.

**Comment by andreymoskvenkov** [comment]
How to check this? 
Do 399.24 drivers mean, that I am using Cuda 9.2 ?

**Comment by funakoshi2718** [comment]
The driver and cuda runtime are 2 different things.

nvcc --version
Displays the Cuda Runtime version.

To install Cuda visit:  https://developer.nvidia.com/cuda-downloads

**Comment by andreymoskvenkov** [comment]
same issue

C:\Users\Rrr1>nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2018 NVIDIA Corporation
Built on Sat_Aug_25_21:08:04_Central_Daylight_Time_2018
Cuda compilation tools, release 10.0, V10.0.130

But the error is a little bit different now:

C:\BitcoinGold\Funakoshi>C:\BitcoinGold\Funakoshi\funakoshiMiner.exe -cd 0 1 2 3 4 5 6 7 --algo 144_5 --pers BgoldPoW -l btg.2miners.com:4040 -u GXd3AQrThEmB5oUDqwcVw798PZ5u2Xrfdc.rig1 --pass x

[2018-Sep-28 11:41:22] [MINER] Using pool: btg.2miners.com
[2018-Sep-28 11:41:22] [MINER] Starting miner
[2018-Sep-28 11:41:22] [GPU-0] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 0)
[2018-Sep-28 11:41:22] [GPU-1] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 1)
[2018-Sep-28 11:41:22] [GPU-2] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 2)
[2018-Sep-28 11:41:22] [GPU-3] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 3)
[2018-Sep-28 11:41:22] [GPU-4] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 4)
[2018-Sep-28 11:41:22] [GPU-5] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 5)
[2018-Sep-28 11:41:22] [GPU-6] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 6)
[2018-Sep-28 11:41:22] [MINER] Connecting to stratum server btg.2miners.com:4040
[2018-Sep-28 11:41:22] [GPU-7] FUNAKOSHI-solver starts mining (device-info: GeForce GTX 1080 Ti, version: 6.1, device: 7)
[2018-Sep-28 11:41:22] [MINER] Connected!
[2018-Sep-28 11:41:22] [MINER] Subscribed to stratum server
[2018-Sep-28 11:41:22] [MINER] Authorized worker GXd3AQrThEmB5oUDqwcVw798PZ5u2Xrfdc.rig1
[2018-Sep-28 11:41:22] [MINER] Target set to 000f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0d
[2018-Sep-28 11:41:22] [MINER] Received new job #886869453628548
nvmlInit failed
[2018-Sep-28 11:41:23] [GPU-4] CUDA: failed to alloc memory


Note: during the installation of CUDA driver I unchecked "Visual Studio Integration" checkbox in drivers components, because insallator has thrown an warning, that Visual Studio components will not work properly due to not installed Visual Studio on my computer.

**Comment by andreymoskvenkov** [comment]
Additional note: I don't have monitor emulator on my farm and don't have monitor too. I am connecting to it, using TeamViewer.

**Comment by funakoshi2718** [comment]
I am deeply sorry.
I don't know why you are getting this error (there is more than enough memory in 1080ti).
You are the only user reporting this kind of problem to me.

**Comment by andreymoskvenkov** [comment]
No, tried with renamed files from Nvidia CUDA directory - same error.

**Comment by andreymoskvenkov** [comment]
I have found what was the cause (I actually got the same error, when tried to use Claymore for ETH mining). It was due to my farm has 16 GB of physical DDR4 RAM and for successful mining I should configure Virtual Memory as 2x of physycal RAM. So I should configure it as at least "33000 MB" in Windows, but I had 24000 MB there. This is because in all instructions people usually have 8 GB of RAM on their farms and they usually suggested to have Virtual RAM as 16-24 GB. So, I set there 24 GB, thinking, that it is enough, but for 16 GB of installed RAM it was not enough :)
Now everything works! However your miner is slower, than EWFB's 0.6 :( 
Your miner shows for my 8 cards total hashrate: 512-520, however EWFB's shows me 610-625 for BTG.
